tests/sles4sap/publiccloud/cluster_add_repos.pm

```diff
@@ -23,6 +23,10 @@ sub run {
 
     while (defined(my $maintrepo = shift @repos)) {
         next if $maintrepo =~ /^\s*$/;
+        if ($maintrepo =~ /Development-Tools/) {
+            record_info("MISSING REPOS", "There are Development-Tools repos in this incident, that are not uploaded to IBSM. Later errors, if they occur, may be due to these.");
```

Comment:
```suggestion
            record_info("FILTER DEVEL REPOS", "There are Development-Tools repos in this incident, that are not uploaded to IBSM. Later errors, if they occur, may be due to these.");
```

Reply:
My idea here is that the frame should stand out to the reviewer, so that they know something's wrong. This will output `MISSING` in the UI, while `FILTER REPOS` may not stand out that much as "something is not right". I even thought of making this `WARNING`, since explanation is given on the detail text, just to grab attention